simpleline/render/prompt.py

```diff
@@ -35,19 +35,19 @@ class Prompt():
     ENTER = N_("ENTER")
 
     # TRANSLATORS: 'q' to quit
```

Comment:
I'm not entirely sure but I think you could drop these comments too, IIRC they are copied to the translation files, but only if they directly precede the line with string.

Reply:
Fixed.